packages/perseus-score/src/util/answer-types.ts

```diff
@@ -442,17 +461,17 @@ const KhanAnswerTypes = {
 
                 // Numbers with percent signs
                 percent: function (text) {
-                    text = $.trim(text);
+                    text = String(text).trim();
```

Comment:
jQuery's `trim` coerces numbers to strings and apparently our code was banking on that.